Source/SwiftLintFramework/Extensions/String+SwiftLint.swift

```diff
@@ -69,4 +69,16 @@ extension String {
         }
         return nil
     }
+
+    var stringByDeletingLastPathComponent: String {
+        return (self as NSString).stringByDeletingLastPathComponent
+    }
+
+    func stringByAppendingPathComponent(pathComponent: String) -> String {
+        return (self as NSString).stringByAppendingPathComponent(pathComponent)
+    }
+
+    public func absolutePathStandardized() -> String {
```

Comment:
It's possible that we never actually need `absolutePathRepresentation`, and that `stringByStandardizingPath` would be a sufficient replacement.